// @uiw/codemirror-theme-duotone/cjs/index.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.duotoneLight = exports.duotoneDark = void 0;

var _highlight = require("@lezer/highlight");

var _codemirrorThemes = require("@uiw/codemirror-themes");

/**
 * @name duotone
 * @author Bram de Haan
 * by Bram de Haan, adapted from DuoTone themes by Simurai (http://simurai.com/projects/2016/01/01/duotone-themes)
 */
var duotoneLight = (0, _codemirrorThemes.createTheme)({
  theme: 'light',
  settings: {
    background: '#faf8f5',
    foreground: '#b29762',
    caret: '#93abdc',
    selection: '#e3dcce',
    selectionMatch: '#e3dcce',
    gutterBackground: '#faf8f5',
    gutterForeground: '#cdc4b1',
    lineHighlight: '#EFEFEF'
  },
  styles: [{
    tag: [_highlight.tags.comment, _highlight.tags.bracket],
    color: '#b6ad9a'
  }, {
    tag: [_highlight.tags.atom, _highlight.tags.number, _highlight.tags.keyword, _highlight.tags.link, _highlight.tags.attributeName, _highlight.tags.quote],
    color: '#063289'
  }, {
    tag: [_highlight.tags.emphasis, _highlight.tags.heading, _highlight.tags.tagName, _highlight.tags.propertyName, _highlight.tags.variableName],
    color: '#2d2006'
  }, {
    tag: [_highlight.tags.typeName, _highlight.tags.url, _highlight.tags.string],
    color: '#896724'
  }, {
    tag: [_highlight.tags.operator, _highlight.tags.string],
    color: '#1659df'
  }, {
    tag: [_highlight.tags.propertyName],
    color: '#b29762'
  }, {
    tag: [_highlight.tags.unit, _highlight.tags.punctuation],
    color: '#063289'
  }]
});
exports.duotoneLight = duotoneLight;
var duotoneDark = (0, _codemirrorThemes.createTheme)({
  theme: 'dark',
  settings: {
    background: '#2a2734',
    foreground: '#6c6783',
    caret: '#ffad5c',
    selection: 'rgba(255, 255, 255, 0.1)',
    gutterBackground: '#2a2734',
    gutterForeground: '#545167',
    lineHighlight: '#36334280'
  },
  styles: [{
    tag: [_highlight.tags.comment, _highlight.tags.bracket],
    color: '#6c6783'
  }, {
    tag: [_highlight.tags.atom, _highlight.tags.number, _highlight.tags.keyword, _highlight.tags.link, _highlight.tags.attributeName, _highlight.tags.quote],
    color: '#ffcc99'
  }, {
    tag: [_highlight.tags.emphasis, _highlight.tags.heading, _highlight.tags.tagName, _highlight.tags.propertyName, _highlight.tags.className, _highlight.tags.variableName],
    color: '#eeebff'
  }, {
    tag: [_highlight.tags.typeName, _highlight.tags.url],
    color: '#7a63ee'
  }, {
    tag: _highlight.tags.operator,
    color: '#ffad5c'
  }, {
    tag: _highlight.tags.string,
    color: '#ffb870'
  }, {
    tag: [_highlight.tags.propertyName],
    color: '#9a86fd'
  }, {
    tag: [_highlight.tags.unit, _highlight.tags.punctuation],
    color: '#e09142'
  }]
});
exports.duotoneDark = duotoneDark;
