// @uiw/codemirror-theme-androidstudio/cjs/index.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.androidstudio = void 0;

var _highlight = require("@lezer/highlight");

var _codemirrorThemes = require("@uiw/codemirror-themes");

/**
 * @name androidstudio
 */
var androidstudio = (0, _codemirrorThemes.createTheme)({
  theme: 'dark',
  settings: {
    background: '#282b2e',
    foreground: '#a9b7c6',
    caret: '#00FF00',
    selection: '#343739',
    selectionMatch: '#343739',
    lineHighlight: '#343739'
  },
  styles: [{
    tag: [_highlight.tags.keyword, _highlight.tags.deleted, _highlight.tags.className],
    color: '#cc7832'
  }, {
    tag: [_highlight.tags.number, _highlight.tags.literal, _highlight.tags.derefOperator],
    color: '#6897bb'
  }, {
    tag: [_highlight.tags.link, _highlight.tags.variableName],
    color: '#629755'
  }, {
    tag: [_highlight.tags.comment, _highlight.tags.quote],
    color: 'grey'
  }, {
    tag: [_highlight.tags.meta, _highlight.tags.documentMeta],
    color: '#bbb529'
  }, {
    tag: [_highlight.tags.string, _highlight.tags.propertyName, _highlight.tags.attributeValue],
    color: '#6a8759'
  }, {
    tag: [_highlight.tags.heading, _highlight.tags.typeName],
    color: '#ffc66d'
  }, {
    tag: [_highlight.tags.attributeName],
    color: '#a9b7c6'
  }, {
    tag: [_highlight.tags.emphasis],
    fontStyle: 'italic'
  }]
});
exports.androidstudio = androidstudio;
