// @uiw/codemirror-theme-darcula/cjs/index.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.darcula = void 0;

var _highlight = require("@lezer/highlight");

var _codemirrorThemes = require("@uiw/codemirror-themes");

/**
 * @name darcula
 * @author darcula
 * Name: IntelliJ IDEA darcula theme
 * From IntelliJ IDEA by JetBrains
 */
var darcula = (0, _codemirrorThemes.createTheme)({
  theme: 'dark',
  settings: {
    background: '#2B2B2B',
    foreground: '#f8f8f2',
    caret: '#FFFFFF',
    selection: 'rgba(255, 255, 255, 0.1)',
    selectionMatch: 'rgba(255, 255, 255, 0.2)',
    gutterBackground: 'rgba(255, 255, 255, 0.1)',
    gutterForeground: '#999',
    lineHighlight: 'rgba(255, 255, 255, 0.1)'
  },
  styles: [{
    tag: [_highlight.tags.atom, _highlight.tags.number],
    color: '#bd93f9'
  }, {
    tag: [_highlight.tags.comment],
    color: '#61A151'
  }, {
    tag: [_highlight.tags.string],
    color: '#6A8759'
  }, {
    tag: [_highlight.tags.variableName, _highlight.tags.operator],
    color: '#A9B7C6'
  }, {
    tag: [_highlight.tags.meta, _highlight.tags.className],
    color: '#A9B7C6'
  }, {
    tag: [_highlight.tags.propertyName],
    color: '#FFC66D'
  }, {
    tag: [_highlight.tags.keyword],
    color: '#CC7832'
  }, {
    tag: [_highlight.tags.tagName],
    color: '#ff79c6'
  }, {
    tag: [_highlight.tags.typeName],
    color: '#ffb86c'
  }]
});
exports.darcula = darcula;
