// @uiw/codemirror-theme-eclipse/cjs/index.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.eclipse = void 0;

var _highlight = require("@lezer/highlight");

var _codemirrorThemes = require("@uiw/codemirror-themes");

var eclipse = (0, _codemirrorThemes.createTheme)({
  theme: 'light',
  settings: {
    background: '#fff',
    foreground: '#000',
    caret: '#FFFFFF',
    selection: '#d7d4f0',
    selectionMatch: '#d7d4f0',
    gutterBackground: '#f7f7f7',
    gutterForeground: '#999',
    lineHighlight: '#e8f2ff'
  },
  styles: [{
    tag: [_highlight.tags.comment],
    color: '#3F7F5F'
  }, {
    tag: [_highlight.tags.documentMeta],
    color: '#FF1717'
  }, {
    tag: _highlight.tags.keyword,
    color: '#7F0055',
    fontWeight: 'bold'
  }, {
    tag: _highlight.tags.atom,
    color: '#00f'
  }, {
    tag: _highlight.tags.number,
    color: '#164'
  }, {
    tag: _highlight.tags.propertyName,
    color: '#164'
  }, {
    tag: [_highlight.tags.variableName, _highlight.tags.definition(_highlight.tags.variableName)],
    color: '#0000C0'
  }, {
    tag: _highlight.tags["function"](_highlight.tags.variableName),
    color: '#0000C0'
  }, {
    tag: _highlight.tags.string,
    color: '#2A00FF'
  }, {
    tag: _highlight.tags.operator,
    color: 'black'
  }, {
    tag: _highlight.tags.tagName,
    color: '#170'
  }, {
    tag: _highlight.tags.attributeName,
    color: '#00c'
  }, {
    tag: _highlight.tags.link,
    color: '#219'
  }]
});
exports.eclipse = eclipse;
