// @uiw/codemirror-theme-sublime/cjs/index.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.sublime = void 0;

var _highlight = require("@lezer/highlight");

var _codemirrorThemes = require("@uiw/codemirror-themes");

var sublime = (0, _codemirrorThemes.createTheme)({
  theme: 'dark',
  settings: {
    background: '#303841',
    foreground: '#FFFFFF',
    caret: '#FBAC52',
    selection: '#4C5964',
    selectionMatch: '#3A546E',
    gutterBackground: '#303841',
    gutterForeground: '#FFFFFF70',
    lineHighlight: '#00000059'
  },
  styles: [{
    tag: [_highlight.tags.meta, _highlight.tags.comment],
    color: '#A2A9B5'
  }, {
    tag: [_highlight.tags.attributeName, _highlight.tags.keyword],
    color: '#B78FBA'
  }, {
    tag: _highlight.tags["function"](_highlight.tags.variableName),
    color: '#5AB0B0'
  }, {
    tag: [_highlight.tags.string, _highlight.tags.regexp, _highlight.tags.attributeValue],
    color: '#99C592'
  }, {
    tag: _highlight.tags.operator,
    color: '#f47954'
  }, // { tag: t.moduleKeyword, color: 'red' },
  {
    tag: [_highlight.tags.propertyName, _highlight.tags.typeName],
    color: '#629ccd'
  }, {
    tag: [_highlight.tags.tagName, _highlight.tags.modifier],
    color: '#E35F63'
  }, {
    tag: [_highlight.tags.number, _highlight.tags.definition(_highlight.tags.tagName), _highlight.tags.className, _highlight.tags.definition(_highlight.tags.variableName)],
    color: '#fbac52'
  }, {
    tag: [_highlight.tags.atom, _highlight.tags.bool, _highlight.tags.special(_highlight.tags.variableName)],
    color: '#E35F63'
  }]
});
exports.sublime = sublime;
