// @uiw/codemirror-theme-github/cjs/index.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.githubLight = exports.githubDark = void 0;

var _highlight = require("@lezer/highlight");

var _codemirrorThemes = require("@uiw/codemirror-themes");

/**
 * @name github
 */
var githubLight = (0, _codemirrorThemes.createTheme)({
  theme: 'light',
  settings: {
    background: '#fff',
    foreground: '#24292e',
    selection: '#BBDFFF',
    selectionMatch: '#BBDFFF',
    gutterBackground: '#fff',
    gutterForeground: '#6e7781'
  },
  styles: [{
    tag: [_highlight.tags.comment, _highlight.tags.bracket],
    color: '#6a737d'
  }, {
    tag: [_highlight.tags.className, _highlight.tags.propertyName],
    color: '#6f42c1'
  }, {
    tag: [_highlight.tags.variableName, _highlight.tags.attributeName, _highlight.tags.number, _highlight.tags.operator],
    color: '#005cc5'
  }, {
    tag: [_highlight.tags.keyword, _highlight.tags.typeName, _highlight.tags.typeOperator, _highlight.tags.typeName],
    color: '#d73a49'
  }, {
    tag: [_highlight.tags.string, _highlight.tags.meta, _highlight.tags.regexp],
    color: '#032f62'
  }, {
    tag: [_highlight.tags.name, _highlight.tags.quote],
    color: '#22863a'
  }, {
    tag: [_highlight.tags.heading],
    color: '#24292e',
    fontWeight: 'bold'
  }, {
    tag: [_highlight.tags.emphasis],
    color: '#24292e',
    fontStyle: 'italic'
  }, {
    tag: [_highlight.tags.deleted],
    color: '#b31d28',
    backgroundColor: 'ffeef0'
  }]
});
exports.githubLight = githubLight;
var githubDark = (0, _codemirrorThemes.createTheme)({
  theme: 'dark',
  settings: {
    background: '#0d1117',
    foreground: '#c9d1d9',
    caret: '#c9d1d9',
    selection: '#003d73',
    selectionMatch: '#003d73',
    lineHighlight: '#36334280'
  },
  styles: [{
    tag: [_highlight.tags.comment, _highlight.tags.bracket],
    color: '#8b949e'
  }, {
    tag: [_highlight.tags.className, _highlight.tags.propertyName],
    color: '#d2a8ff'
  }, {
    tag: [_highlight.tags.variableName, _highlight.tags.attributeName, _highlight.tags.number, _highlight.tags.operator],
    color: '#79c0ff'
  }, {
    tag: [_highlight.tags.keyword, _highlight.tags.typeName, _highlight.tags.typeOperator, _highlight.tags.typeName],
    color: '#ff7b72'
  }, {
    tag: [_highlight.tags.string, _highlight.tags.meta, _highlight.tags.regexp],
    color: '#a5d6ff'
  }, {
    tag: [_highlight.tags.name, _highlight.tags.quote],
    color: '#7ee787'
  }, {
    tag: [_highlight.tags.heading],
    color: '#d2a8ff',
    fontWeight: 'bold'
  }, {
    tag: [_highlight.tags.emphasis],
    color: '#d2a8ff',
    fontStyle: 'italic'
  }, {
    tag: [_highlight.tags.deleted],
    color: '#ffdcd7',
    backgroundColor: 'ffeef0'
  }]
});
exports.githubDark = githubDark;
