// @uiw/codemirror-theme-bbedit/cjs/index.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.bbedit = void 0;

var _highlight = require("@lezer/highlight");

var _codemirrorThemes = require("@uiw/codemirror-themes");

var bbedit = (0, _codemirrorThemes.createTheme)({
  theme: 'light',
  settings: {
    background: '#FFFFFF',
    foreground: '#000000',
    caret: '#FBAC52',
    selection: '#FFD420',
    selectionMatch: '#FFD420',
    gutterBackground: '#f5f5f5',
    gutterForeground: '#4D4D4C',
    lineHighlight: '#00000012'
  },
  styles: [{
    tag: [_highlight.tags.meta, _highlight.tags.comment],
    color: '#804000'
  }, {
    tag: [_highlight.tags.keyword, _highlight.tags.strong],
    color: '#0000FF'
  }, {
    tag: [_highlight.tags.number],
    color: '#FF0080'
  }, {
    tag: [_highlight.tags.string],
    color: '#FF0080'
  }, {
    tag: [_highlight.tags.variableName],
    color: '#006600'
  }, {
    tag: [_highlight.tags.escape],
    color: '#33CC33'
  }, {
    tag: [_highlight.tags.tagName],
    color: '#1C02FF'
  }, {
    tag: [_highlight.tags.heading],
    color: '#0C07FF'
  }, {
    tag: [_highlight.tags.quote],
    color: '#000000'
  }, {
    tag: [_highlight.tags.list],
    color: '#B90690'
  }, {
    tag: [_highlight.tags.documentMeta],
    color: '#888888'
  }, {
    tag: [_highlight.tags["function"](_highlight.tags.variableName)],
    color: '#0000A2'
  }, {
    tag: [_highlight.tags.definition(_highlight.tags.typeName), _highlight.tags.typeName],
    color: '#6D79DE'
  }]
});
exports.bbedit = bbedit;
