// @uiw/codemirror-theme-bespin/cjs/index.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.bespin = void 0;

var _highlight = require("@lezer/highlight");

var _codemirrorThemes = require("@uiw/codemirror-themes");

/**
 * @name Bespin
 * @author Mozilla / Jan T. Sott
 *
 * CodeMirror template by Jan T. Sott (https://github.com/idleberg/base16-codemirror)
 * Original Base16 color scheme by Chris Kempson (https://github.com/chriskempson/base16)
 */
var bespin = (0, _codemirrorThemes.createTheme)({
  theme: 'dark',
  settings: {
    background: '#28211c',
    foreground: '#9d9b97',
    caret: '#797977',
    selection: '#36312e',
    selectionMatch: '#4f382b',
    gutterBackground: '#28211c',
    gutterForeground: '#666666',
    lineHighlight: 'rgba(255, 255, 255, 0.1)'
  },
  styles: [{
    tag: [_highlight.tags.atom, _highlight.tags.number, _highlight.tags.link, _highlight.tags.bool],
    color: '#9b859d'
  }, {
    tag: _highlight.tags.comment,
    color: '#937121'
  }, {
    tag: [_highlight.tags.keyword, _highlight.tags.tagName],
    color: '#cf6a4c'
  }, {
    tag: _highlight.tags.string,
    color: '#f9ee98'
  }, {
    tag: _highlight.tags.bracket,
    color: '#9d9b97'
  }, {
    tag: [_highlight.tags.variableName],
    color: '#5ea6ea'
  }, {
    tag: _highlight.tags.definition(_highlight.tags.variableName),
    color: '#cf7d34'
  }, {
    tag: [_highlight.tags["function"](_highlight.tags.variableName), _highlight.tags.className],
    color: '#cf7d34'
  }, {
    tag: [_highlight.tags.propertyName, _highlight.tags.attributeName],
    color: '#54be0d'
  }]
});
exports.bespin = bespin;
