// @uiw/codemirror-theme-xcode/cjs/index.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.xcodeLight = exports.xcodeDark = void 0;

var _highlight = require("@lezer/highlight");

var _codemirrorThemes = require("@uiw/codemirror-themes");

/**
 * @name github
 */
var xcodeLight = (0, _codemirrorThemes.createTheme)({
  theme: 'light',
  settings: {
    background: '#fff',
    foreground: '#3D3D3D',
    selection: '#BBDFFF',
    selectionMatch: '#BBDFFF',
    gutterBackground: '#fff',
    gutterForeground: '#AFAFAF',
    lineHighlight: '#EDF4FF'
  },
  styles: [{
    tag: [_highlight.tags.comment, _highlight.tags.quote],
    color: '#707F8D'
  }, {
    tag: [_highlight.tags.typeName, _highlight.tags.typeOperator],
    color: '#aa0d91'
  }, {
    tag: [_highlight.tags.keyword],
    color: '#aa0d91',
    fontWeight: 'bold'
  }, {
    tag: [_highlight.tags.string, _highlight.tags.meta],
    color: '#D23423'
  }, {
    tag: [_highlight.tags.name],
    color: '#032f62'
  }, {
    tag: [_highlight.tags.typeName],
    color: '#522BB2'
  }, {
    tag: [_highlight.tags.variableName],
    color: '#23575C'
  }, {
    tag: [_highlight.tags.definition(_highlight.tags.variableName)],
    color: '#327A9E'
  }, {
    tag: [_highlight.tags.regexp, _highlight.tags.link],
    color: '#0e0eff'
  }]
});
exports.xcodeLight = xcodeLight;
var xcodeDark = (0, _codemirrorThemes.createTheme)({
  theme: 'dark',
  settings: {
    background: '#292A30',
    foreground: '#CECFD0',
    caret: '#fff',
    selection: '#727377',
    selectionMatch: '#727377',
    lineHighlight: '#2F3239'
  },
  styles: [{
    tag: [_highlight.tags.comment, _highlight.tags.quote],
    color: '#7F8C98'
  }, {
    tag: [_highlight.tags.keyword],
    color: '#FF7AB2',
    fontWeight: 'bold'
  }, {
    tag: [_highlight.tags.string, _highlight.tags.meta],
    color: '#FF8170'
  }, {
    tag: [_highlight.tags.typeName],
    color: '#DABAFF'
  }, {
    tag: [_highlight.tags.definition(_highlight.tags.variableName)],
    color: '#6BDFFF'
  }, {
    tag: [_highlight.tags.name],
    color: '#6BAA9F'
  }, {
    tag: [_highlight.tags.variableName],
    color: '#ACF2E4'
  }, {
    tag: [_highlight.tags.regexp, _highlight.tags.link],
    color: '#FF8170'
  }]
});
exports.xcodeDark = xcodeDark;
var test = /^goode/;
