// @uiw/codemirror-theme-atomone/cjs/index.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.atomone = void 0;

var _highlight = require("@lezer/highlight");

var _codemirrorThemes = require("@uiw/codemirror-themes");

/**
 * @name Atom One
 * Atom One dark syntax theme
 *
 * https://github.com/atom/one-dark-syntax
 */
var atomone = (0, _codemirrorThemes.createTheme)({
  theme: 'dark',
  settings: {
    background: '#272C35',
    foreground: '#9d9b97',
    caret: '#797977',
    selection: '#ffffff30',
    selectionMatch: '#2B323D',
    gutterBackground: '#272C35',
    gutterForeground: '#465063',
    lineHighlight: '#2B323D'
  },
  styles: [{
    tag: [_highlight.tags["function"](_highlight.tags.variableName), _highlight.tags["function"](_highlight.tags.propertyName), _highlight.tags.url, _highlight.tags.processingInstruction],
    color: 'hsl(207, 82%, 66%)'
  }, {
    tag: [_highlight.tags.tagName, _highlight.tags.heading],
    color: '#e06c75'
  }, {
    tag: _highlight.tags.comment,
    color: '#54636D'
  }, {
    tag: [_highlight.tags.propertyName],
    color: 'hsl(220, 14%, 71%)'
  }, {
    tag: [_highlight.tags.attributeName, _highlight.tags.number],
    color: 'hsl( 29, 54%, 61%)'
  }, {
    tag: _highlight.tags.className,
    color: 'hsl( 39, 67%, 69%)'
  }, {
    tag: _highlight.tags.keyword,
    color: 'hsl(286, 60%, 67%)'
  }, {
    tag: [_highlight.tags.string, _highlight.tags.regexp, _highlight.tags.special(_highlight.tags.propertyName)],
    color: '#98c379'
  }]
});
exports.atomone = atomone;
